package-build.yml

```diff
@@ -22,7 +22,7 @@ spec:
               - .imgpkg/images.yml
       export:
       - imgpkgBundle:
-          image: ghcr.io/carvel-dev/kapp-controller-package-bundle
+          image: ghcr.io/rcmadhankumar/kapp-controller-package-bundle
```

Comment:
I think this was intended for testing on your fork?

Reply:
Yes. this will be changed. will resolve this once the change is done.